src/main/java/org/fibsters/CoordinatorComputeEngineImpl.java

```diff
@@ -79,11 +79,14 @@ public void payloadPoolLoop() {
     @Override
     public ComputeJob createComputeJobFromInputPayload(InputPayloadImpl inputPayload) {
         OutputPayloadImpl outputPayload = new OutputPayloadImpl(0, inputPayload, ComputeJobStatus.UNSTARTED);
-        FibCalcComputeEngineImpl fibCalcCE = new FibCalcComputeEngineImpl(outputPayload);
+        MultipartComputeJob computeJob = new MultipartComputeJob(outputPayload);
```

Comment:
We now assume a multipart job always. Ideally, we'd give the user a choice to compute a spiral or not.